azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -67,51 +60,65 @@
 import java.nio.file.Paths;
 import java.security.KeyStore;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import java.util.zip.ZipFile;
-import javax.sql.DataSource;
-import org.apache.commons.dbutils.QueryRunner;
-import org.apache.log4j.Logger;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- *  This class is responsible for launching a flow in a container.
+ *  This class is the entrypoint for launching a flow execution in a container.
+ *  It sets up the Azkaban Properties, the DAO, injects all the required classes, sets up the
+ *  execution directory along with the project files, creates the FlowRunner and submits it to
+ *  the executor service for execution.
+ *  It assumes that their is a certain directory structure consisting of all the dependencies.
```

Comment:
Can you please mention what directory structure it is assuming in comment? The exact path or any that it is expecting?